src/smolagents/gradio_ui.py

```diff
@@ -188,15 +188,26 @@ def __init__(self, agent: MultiStepAgent, file_upload_folder: str | None = None)
             if not os.path.exists(file_upload_folder):
                 os.mkdir(file_upload_folder)
 
```

Comment:
This change will allow people to use their agent's name and description in the UI:
```suggestion
self.name = getattr(agent, "name", None)
self.description = getattr(agent, "description", None)
```